React/Views/RCTViewManager.m

```diff
@@ -259,6 +260,21 @@ - (RCTShadowView *)shadowView
     view.removeClippedSubviews = json ? [RCTConvert BOOL:json] : defaultView.removeClippedSubviews;
   }
 }
+RCT_CUSTOM_VIEW_PROPERTY(borderCurve, RCTBorderCurve, RCTView)
+{
+  if (@available(iOS 13.0, *)) {
+    switch ([RCTConvert RCTBorderCurve:json]) {
+      case RCTBorderCurveContinuous:
+        view.layer.cornerCurve = kCACornerCurveContinuous;
+        break;
+      case RCTBorderCurveCircular:
+        view.layer.cornerCurve = kCACornerCurveCircular;
+        break;
+      default:
+        view.layer.cornerCurve = kCACornerCurveCircular;
```

Comment:
we don't need this - since RCTBorderCurve: returns a RCTBorderCurve enum value, we're protected by the compiler.

default can be risky because this callsite won't be protected if there are new values added to this enum.

Reply:
Agreed, done